src/Bookings.js

```diff
@@ -13,6 +22,19 @@ const Bookings = () => {
       <div className="container">
         <Search search={search} />
         {/* <SearchResults results={FakeBookings} /> */}
+        <SearchResults results={FakeBookings} />
+       
+        
+        <TouristInfoCards  town="Visit Glasgow"  link = "http://peoplemakeglasgow.com" imageSource = "https://peoplemakeglasgow.com/imager/general/109009/City-centre-George-Square-Content-Block_aa93d8ee9a3cc573b9a01b328410e8b2.jpg"/>
+        <TouristInfoCards town = "Visit Manchester" link = "http://visitmanchester.com" imageSource = "https://media.wired.co.uk/photos/6357b98e09e6f1942a2a8a9a/16:9/w_1280,c_limit/HSBC%20MANCHESTER.jpeg"/>
+        <TouristInfoCards town = "Visit London" link = "http://visitlondon.com" imageSource = "https://cdn.londonandpartners.com/-/media/images/london/visit/campaigns/international-recovery-campaign/lets-do-london-related-international-640.jpeg?mw=640&hash=477E6FF9F31F292EE017E5BA63027C7054EB39A0"/>
```

Comment:
Really nice use of a component here to reduce HTML repetitiveness. One small change you could make is to make this information an array of objects, and then `.map` over that array to render out the results. This means if you extend this to get this information from an API, or want to add a new town, you just need to add a row to that array rather than manually write more JSX. e.g.
```
const townArr = [
   {titleText: 'Visit Glasgow', link: 'http://...', imgLink: 'http://...'},
   {titleText: 'Visit Manchester', link: 'http://...', imgLink: 'http://...'},
]

const  Bookings  = () => {
   ...

   return (
      ...
      { townArr.map(({titleText, link, imgLink}) => <TouristInfoCard key={titleText} town={titleText} link={link} img={imgLink}>) }
   )
}

```